### ISSUE TITLE
Misleading warning while creating a ClientSession

### ISSUE THREAD

**Issue by cpvandehey** [fafhrd91/aiohttp#7262]
### Describe the bug

Howdy all!

[I suggest rewording this warning](https://github.com/aio-libs/aiohttp/blob/33953f110e97eecc707e1402daa8d543f38a189b/aiohttp/helpers.py#L291)

IMO, this is misleading because a `ClientSession` can and (_at least in my experience_) probably should exist within a synchronous function **if** you're using it outside of an async context manager. The creation of the object is NOT asynchronous in any way and it is only the "closing" of it that needs to be awaited.

This warning should probably say something like, "The object should be created within a running event loop". This would be a much more clear warning for the actual problem that could arise from misuse.

### To Reproduce

n/a

### Expected behavior

n/a

### Logs/tracebacks

```python-traceback
n/a
```

### Python Version

```console
n/a
```

### aiohttp Version

```console
n/a
```

### multidict Version

```console
n/a
```

### yarl Version

```console
n/a
```

### OS

n/a

### Related component

Server, Client

### Additional context

n/a

### Code of Conduct

- [X] I agree to follow the aio-libs Code of Conduct

**Comment by Dreamsorcerer** [comment]
Code no longer exists. It was a compatibility stub for Python 3.6.
